qa/rpc-tests/mempool_size_limit_even_more.py

```diff
@@ -0,0 +1,439 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.test_framework import ForkHeights
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import initialize_chain_clean, start_nodes, connect_nodes_bi, mark_logs,\
+    get_epoch_data, swap_bytes, assert_greater_than, colorize as cc
+from test_framework.mc_test.mc_test import *
+import os
+import zipfile
+from decimal import Decimal, ROUND_DOWN
+
+DEBUG_MODE = 1
+NUMB_OF_NODES = 2
+EPOCH_LENGTH = 100
+
+class feeAmounts():
+    HIGHEST = Decimal('0.0001')
+    HIGH    = Decimal('0.00005')
+    MEDIUM  = Decimal('0.00001')
+    LOW     = Decimal('0.000005')
+    LOWEST  = Decimal('0.000001')
+    NULL    = Decimal('0')
+
+FT_SC_FEE      = feeAmounts.NULL
+MBTR_SC_FEE    = feeAmounts.NULL
+CERT_FEE       = Decimal('0.00015')
+
+NODE0_LIMIT_B = 4000000
+NODE1_LIMIT_B = 4000000
+
+NODE0_CERT_LIMIT_B = NODE0_LIMIT_B / 2
+NODE1_CERT_LIMIT_B = NODE1_LIMIT_B / 2
+
+EPSILON = 100000
+MAX_FEE = Decimal("999999")
+
+NUM_CEASING_SIDECHAINS    = 750
+
+PARAMS_NAME = "sc"
+USE_SNAPSHOT = True
+
+"""
+This test is a follow up of mempool_size_limit.py and mempool_size_limit_more.py, and uses a new snapshot data.
+Snapshot creation takes slightly more than an hour, and the resulting archive will be about 2.5 GBytes in size.
+This test is currently DISABLED from the CI, until we find a better option for storing and retrieving the
+archive.
+
+Test MAX_SIZE for certificates
+    Clear the mempool
+    Add certificates up to the max size, all related to different sidechains (no duplications), fee_rate = F_medium [OK]
+        Add one more certificate exceeding the max size, fee_rate = F_low [REJECT]
+        Add one more certificate exceeding the max size, fee_rate = F_high [ACCEPT, one certificate gets evicted]
+    Add transactions up to MAX_SIZE / 2, fee_rate = F_highest [ACCEPT, certificates get evicted]
+        Add one more transaction, fee_rate = F_highest [REJECTED, or ACCEPTED with the eviction of another transaction]
```

Comment:
Please rephrase this line keeping only `Add one more transaction, fee_rate = F_highest [REJECTED]` since the test flow expects always a rejection.

Reply:
Yes, you are right! Corrected.